### PR TITLE
[IMP][14.0] pms refact folio tests

### PR DESCRIPTION
**Pull request by DarioLodeiros**
- Test Folio cases standarization 
- Change default checkin/out by compute method (to allowed create reservation with specific reservation_line_ids)
- Add constrain consecutive dates on reservation

### CODE REVIEW

File: pms/tests/test_pms_folio_invoice.py [hunk 1]
```diff
@@ -208,7 +202,6 @@ def test_invoice_partial_folio_diferent_partners(self):
 
     def test_invoice_partial_folio_wrong_qtys(self):
         # ARRANGE
```
Comment (miguelpadin) [inline pms/tests/test_pms_folio_invoice.py:204]: a short descriptive text of the task of the test can help a lot to understand what is being tested

File: pms/tests/test_pms_folio_invoice.py [hunk 2]
```diff
@@ -162,7 +157,6 @@ def test_invoice_partial_folio_by_steps(self):
 
     def test_invoice_partial_folio_diferent_partners(self):
         # ARRANGE
```
Comment (miguelpadin) [inline pms/tests/test_pms_folio_invoice.py:159]: a short descriptive text of the task of the test can help a lot to understand what is being tested

File: pms/tests/test_pms_folio_invoice.py [hunk 3]
```diff
@@ -121,7 +117,6 @@ def test_invoice_full_folio(self):
 
     def test_invoice_partial_folio_by_steps(self):
         # ARRANGE
```
Comment (miguelpadin) [inline pms/tests/test_pms_folio_invoice.py:119]: a short descriptive text of the task of the test can help a lot to understand what is being tested

File: pms/tests/test_pms_folio_invoice.py [hunk 4]
```diff
@@ -95,7 +92,6 @@ def create_common_scenario(self):
 
     def test_invoice_full_folio(self):
         # ARRANGE
```
Comment (miguelpadin) [inline pms/tests/test_pms_folio_invoice.py:94]: a short descriptive text of the task of the test can help a lot to understand what is being tested

File: pms/tests/test_pms_folio_sale_line.py [hunk 5]
```diff
@@ -1,43 +1,31 @@
 import datetime
```
Comment (miguelpadin) [inline pms/tests/test_pms_folio_sale_line.py:2]: Review texts at the begining of some tests. Should be a docstring

**Review by miguelpadin — CHANGES_REQUESTED** [review]

File: pms/tests/test_pms_folio_invoice.py (hunk 3 above)
Comment (DarioLodeiros) [reply pms/tests/test_pms_folio_invoice.py:119]: folio invoice test cases refactoring not included int thi PR, we will include them in another PR to be able to activate the changes of this

**Pull request merged**